File: src/gestorAplicacion/personas/Persona.java
/* Autores: Diego Andres Gracia Granados, Daniel Giraldo Vanegas, Elian David Velandia Riveros, Juan Camilo Gutierrez Martinez
y Santiago Arboleda Acevedo */
package gestorAplicacion.personas;

import java.io.Serializable;

//Clase destinada para que hereden doctor y persona
public class Persona implements Serializable {
    //Atributos
	private int cedula;
    private String nombre;
    private String tipoEps;

    //Constructor
    public Persona(int cedula, String nombre, String tipoEps) {
    	this.cedula = cedula;
    	this.nombre = nombre;
    	this.tipoEps = tipoEps;
    }

    //Getters y Setters
    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    
    public String getTipoEps() {
        return tipoEps;
    }

    public void setTipoEps(String tipoEps) {
        this.tipoEps = tipoEps;
    }

    public String bienvenida(){
        return "Bienvenido Usuario "+nombre;
    }
}

File: Python/src/gestor_aplicacion/personas/persona.py
# Autores: Diego Andres Gracia Granados, Daniel Giraldo Vanegas, Elian David Velandia Riveros, Juan Camilo Gutierrez Martinez
# y Santiago Arboleda Acevedo

# Clase destinada para que hereden doctor y persona
class Persona:

    # Atributos y constructor
    def __init__(self, cedula, nombre, tipo_eps):
        self._cedula = cedula
        self._nombre = nombre
        self._tipo_eps=tipo_eps


    @property
    def cedula(self):
        return self._cedula

    @property
    def nombre(self):
        return self._nombre

    @nombre.setter
    def nombre(self, value):
        self._nombre = value

    @property
    def tipo_eps(self):
        return self._tipo_eps
